app/api.py

```diff
@@ -166,9 +166,13 @@ async def create_mentor(data: Mentor):
     Args:
         data (Mentor): Mentor
     Returns:
-        New record data or schema discrepancy error as dictionary
+        New record data or schema discrepancy error as dictionary6
     """
-    return {"result": API.db.create("Mentors", data.dict())}
+    try:
+        return {"result": API.db.create("Mentors", data.dict())}
+    except:
```

Comment:
I like the HTTPException as the return value, great job there.

Naked exceptions should be avoided. What exception are we trying to catch, let's only catch that one to avoid having an api that lies to our users. Check the example.